SlotItem/SlotItem.module.less

```diff
@@ -12,7 +12,7 @@
 
 .slotItem {
 	// Apply to the marquee content area to tell it to expand to the available width of the container
-	.content {
+	.marquee {
```

Comment:
😮 Is this changed from `content` to `marquee` on ui/MarqueeBase?

Reply:
the public class name for marquee is `marquee`. so I don't need to change it on ui/MarqueeBase